README.md

```diff
@@ -63,15 +63,59 @@ Then add `__tests__` to `sources` in your `bsconfig.json`:
 ```js
 "sources": [
   {
-    "dir": "src"
+    "dir": "app",
+    "subdirs": true
```

Comment:
Having this here is a bit opinionated I think, an kind of implies it's necessary for bs-jest to work, when it really isn't.